### DISCUSSION TITLE
Bagaimana cara menambahkan AdBlockID di Brave Browser Android?

### DISCUSSION THREAD

**Discussion by Jokopentil** [realodix/AdBlockID#1346] [Q&A] [answered]
- Bagaimana cara menambahkan AdBlockID di Brave Browser Android?
- Apakah bisa secara manual atau langsung?

**Comment by realodix** [comment]
Judulnya Saya edit, dan pertanyaannya Saya tambahkan juga di bagian deskripsi.

Apakah ini sudah sesuai? Anda bisa mengeditnya jika dirasa belum sesuai.

Jika sudah, nanti akan Saya jawab dan pindahkan ke [Discussions](https://github.com/realodix/AdBlockID/discussions) agar orang lain yang memiliki pertanyaan yang sama dapat dengan mudah menemukannya.

**Comment by realodix** [comment, marked as answer]
> Apakah bisa secara manual atau langsung?
Saat ini harus manual

> Bagaimana cara menambahkan AdBlockID di Brave Browser Android?

Ini relatif mudah
- Buka `brave://adblock` atau `chrome://adblock`
- Lalu klik `Add filter list via URL`
- Lalu masukin deh URL filter untuk AdBlockID (ambil di halaman depan)

Tapi ini benar-benar TIDAK DISARANKAN. Saat ini AdBlockID hanya mendukung AdGuard dan uBlock Origin, menggunakan AdBlockID di luar adblocker tersebut dapat menyebabkan penurunan performa pada AdBlockID itu sendiri.

Terakhir kali mencoba di brave, ada cukup banyak fitur AdBlockID yang bekerja dengan baik di AdGuard dan uBlock Origin, namun tidak berjalan di Brave.
